<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Formulario</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Ver Informacion</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="columnCount">
           <number>4</number>
          </property>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Otros</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>La cuenta va en : 0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="boton_con_icono">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Boton con Icono</string>
        </property>
        <property name="icon">
         <iconset resource="archivos.qrc">
          <normaloff>:/imagenes/files_folder_256.png</normaloff>:/imagenes/files_folder_256.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="boton2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>130</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Boton 2</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionFormulario"/>
    <addaction name="actionVer_Informacion"/>
    <addaction name="actionOtros"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <action name="actionFormulario">
   <property name="text">
    <string>Formulario</string>
   </property>
  </action>
  <action name="actionVer_Informacion">
   <property name="text">
    <string>Ver Informacion</string>
   </property>
  </action>
  <action name="actionOtros">
   <property name="text">
    <string>Otros</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="archivos.qrc"/>
 </resources>
 <connections/>
</ui>
